consoles/sshVirtshSUT.pm

```diff
@@ -54,7 +63,8 @@ sub activate {
     my ($self) = @_;
 
     my $backend = $self->{backend};
-    my ($ssh, $chan) = $backend->open_serial_console_via_ssh($self->{libvirt_domain}, $self->{serial_port_no});
+    my ($ssh, $chan) = $backend->open_serial_console_via_ssh($self->{libvirt_domain},
+        devname => $self->{pty_dev}, port => $self->{serial_port_no}, is_terminal => 1);
 
     $self->{ssh}    = $ssh;
     $self->{screen} = consoles::virtio_screen->new($chan, $ssh->sock);
```

Comment:
I thought @cfconrad or someone changed the name of `virtio_screen`? Seeing as we are now using it for SSH and other protocols

Reply:
I never finished that ssh-backend thingy. Thats why it is still virtio_screen.

Reply:
I renamed it in earlier version if this PR. But then I remove this commit as I didn't think it brings any value. But if you want I can add it again.

Reply:
It should probably be in a separate PR. 

It is just a bit confusing as it implies you are using VIRTIO with an SSH port.

Reply:
I'll add it again than, as a last commit.

Reply:
Module renamed.